libs/google-analytics/api/src/lib/google-analytics.service.ts

```diff
@@ -71,16 +71,25 @@ export class GoogleAnalyticsService implements HotAndTrendingDataSource {
         return object
       }
 
+      // Don't include sub pages of the article prefix
+      if (slug?.split('/').length !== this.config.articlePrefix.split('/').length) {
```

Comment:
hides urls like `/a/tag/news` in case they ever have more views than the articles themselves etc.